DependencyInjection/Configuration.php

```diff
@@ -91,7 +91,11 @@ private function addDbalSection(ArrayNodeDefinition $node) : void
                             ->end()
                             ->children()
                                 ->scalarNode('class')->isRequired()->end()
-                                ->booleanNode('commented')->setDeprecated()->end()
+                                ->booleanNode('commented')->setDeprecated(
+                                    'doctrine/doctrine-bundle',
```

Comment:
With `symfony/config` < 5.1 this change would now mean that the message displayed to the user would be `doctrine/doctrine-bundle` as the only argument of `setDeprecated()` used to be the message.

Reply:
hmm, would something along the lines of 
```
$messageArray = $this-getMessage();
private func getMessage(): array
{
 if config < 5.1 $message = [] else $message ['doctrine/bundle', 2.0, 'this is why';
}

....->setDeprecated(...$message)->end()
```
do the trick?

Reply:
Yes, I think so.

Reply:
i believe this is now taken care of. See my comment below...